coding/react.md

```diff
@@ -4,15 +4,32 @@ layout: base
 
 # React coding standards
 
-This document outlines our standards for writing React applications.
+This document outlines our standards for writing React applications and components.
 
-## Starter Project
+## Starting a new React Project
+When starting a new react application, we recommend [create-react-app](https://github.com/facebook/create-react-app) for bootstrapping the project. Avoid [ejecting](https://facebook.github.io/create-react-app/docs/available-scripts#npm-run-eject) as long as feasible to make updating dependencies easier.
 
-[canonical-webteam/react-starter](https://github.com/canonical-webteam/react-starter) provides a codified
-example of our standards for react projects (Prettier, AirBnB style).
+### Preferred libraries
+All projects should generally use the following:
 
-As much as possible, this project should reflect our standards for React applications. Please make pull requests
-to this project if you feel something should change.
+* component testing - [enzyme](https://github.com/airbnb/enzyme)
+* code formatting - [prettier](https://prettier.io) - We realise that not everyone will agree on the choices that Prettier makes, however feel consistency is ultimately more valuable. As has been said in the golang community, "gofmt's style is no one's favorite, yet gofmt is everyone's favorite".
```

Comment:
Mentioning code formatting and linting here separately kind of suggests that it may be different from our other standards. React project is a JS project, so it should follow our general JavaScript and code formatting practices. Both prettier and linting are mentioned on [code formatting](https://canonical-webteam.github.io/practices/coding/code-formatting) page.

I don't mind listing these libraries here again (although it is a bit of a duplication that can lead to us having couple of places to update), but it would be good to at least link from here to https://canonical-webteam.github.io/practices/coding/code-formatting and https://canonical-webteam.github.io/practices/coding/javascript

Reply:
Good suggestion, thanks @bartaz - have removed references to eslint and prettier, and now link to `code-formatting`.